perception/autoware_tensorrt_bevdet/src/bevdet_node.cpp

```diff
@@ -0,0 +1,235 @@
+  std::string engine_file_base =
+    this->declare_parameter<std::string>("engine_path", "bevdet_one_lt_d");
+  engine_file_ = engine_file_base + (precision_ == "fp16" ? "_fp16.engine" : "_fp32.engine");
+
+  imgs_name_ = this->declare_parameter<std::vector<std::string>>("data_params.cams");
+  class_names_ =
+    this->declare_parameter<std::vector<std::string>>("post_process_params.class_names");
+
+  RCLCPP_INFO_STREAM(this->get_logger(), "Successful load config!");
+
+  sampleData_.param = camParams(cams_intrin_, cams2ego_rot_, cams2ego_trans_);
+
+  RCLCPP_INFO_STREAM(this->get_logger(), "Successful load image params!");
+
+  bevdet_ = std::make_shared<BEVDet>(
+    model_config_, img_N_, sampleData_.param.cams_intrin, sampleData_.param.cams2ego_rot,
+    sampleData_.param.cams2ego_trans, onnx_file_, engine_file_, precision_);
+
+  RCLCPP_INFO_STREAM(this->get_logger(), "Successful create bevdet!");
+
+  CHECK_CUDA(cudaMalloc(
+    reinterpret_cast<void **>(&imgs_dev_), img_N_ * 3 * img_w_ * img_h_ * sizeof(uchar)));
+
+  pub_boxes_ = this->create_publisher<autoware_perception_msgs::msg::DetectedObjects>(
+    "~/output/boxes", rclcpp::QoS{1});
+}
+
+void TRTBEVDetNode::checkInitialization()
+{
+  if (camera_info_received_flag_) {
+    RCLCPP_INFO_STREAM(
+      this->get_logger(), "Camera Info and TF Transform Initialization completed!");
+    initModel();
+    startImageSubscription();
+    timer_->cancel();
+    timer_.reset();
+  } else {
+    RCLCPP_INFO_THROTTLE(
+      this->get_logger(), *this->get_clock(), 5000,
+      "Waiting for Camera Info and TF Transform Initialization...");
+  }
+}
+
+void TRTBEVDetNode::startImageSubscription()
+{
+  using std::placeholders::_1;
+  using std::placeholders::_2;
+  using std::placeholders::_3;
+  using std::placeholders::_4;
+  using std::placeholders::_5;
+  using std::placeholders::_6;
+
+  sub_f_img_.subscribe(this, "~/input/topic_img_f", rclcpp::QoS{1}.get_rmw_qos_profile());
+  sub_b_img_.subscribe(this, "~/input/topic_img_b", rclcpp::QoS{1}.get_rmw_qos_profile());
+
+  sub_fl_img_.subscribe(this, "~/input/topic_img_fl", rclcpp::QoS{1}.get_rmw_qos_profile());
+  sub_fr_img_.subscribe(this, "~/input/topic_img_fr", rclcpp::QoS{1}.get_rmw_qos_profile());
+
+  sub_bl_img_.subscribe(this, "~/input/topic_img_bl", rclcpp::QoS{1}.get_rmw_qos_profile());
+  sub_br_img_.subscribe(this, "~/input/topic_img_br", rclcpp::QoS{1}.get_rmw_qos_profile());
+
+  sync_ = std::make_shared<Sync>(
+    MySyncPolicy(10), sub_fl_img_, sub_f_img_, sub_fr_img_, sub_bl_img_, sub_b_img_, sub_br_img_);
+
+  sync_->registerCallback(std::bind(&TRTBEVDetNode::callback, this, _1, _2, _3, _4, _5, _6));
+}
+
+void TRTBEVDetNode::startCameraInfoSubscription()
+{
+  // cams: ["CAM_FRONT_LEFT", "CAM_FRONT", "CAM_FRONT_RIGHT", "CAM_BACK_LEFT", "CAM_BACK",
+  // "CAM_BACK_RIGHT"]
+  sub_fl_caminfo_ = this->create_subscription<sensor_msgs::msg::CameraInfo>(
+    "~/input/topic_img_fl/camera_info", rclcpp::QoS{1},
```

Comment:
Could change the QoS to something like "rclcpp::SensorDataQoS{}.keep_last(1)"?
I can't received the camera info within the autoware (it was possible with your ros2_dataset_bridge).